src/sonic-config-engine/config_samples.py

```diff
@@ -15,10 +15,10 @@ def generate_t1_sample_config(data):
     for port in natsorted(data['PORT']):
         data['PORT'][port]['admin_status'] = 'up'
         data['PORT'][port]['mtu'] = '9100'
-        local_addr = '10.0.{}.{}'.format(2 * port_count / 256, 2 * port_count % 256)
-        peer_addr = '10.0.{}.{}'.format(2 * port_count / 256, 2 * port_count % 256 + 1)
-        peer_name='ARISTA{0:02d}{1}'.format(1+port_count%(total_port_amount/2), 'T2' if port_count < (total_port_amount/2) else 'T0')
-        peer_asn = 65200 if port_count < total_port_amount/2 else 64001 + port_count - total_port_amount/2
+        local_addr = '10.0.{}.{}'.format(2 * port_count // 256, 2 * port_count % 256)
```

Comment:
@jleveque, curious what the issue was here. Was any of the dividend float?

Reply:
Yes. And the failure that exposed it was occurring due to the `{0:02d}` formatter on line 20, which is expecting an int but was receiving a float.